### PR TITLE
feat: fix robots.txt

### PR DESCRIPTION
**Pull request by anglepoised**
* repair syntax
* ~~disallow all `/edit` paths, not just the root~~

### CODE REVIEW

File: public/robots.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # robots.txt for measured.co
 
 User-agent: *
-Disallow /edit/
+Disallow: */edit
```
Comment (chrisvxd) [inline public/robots.txt:4]: Hm I'm not sure we need this rule at all because `/edit` will 404 if the user is not logged in

Comment (chrisvxd) [reply public/robots.txt:4]: Not that it hurts.

Comment (anglepoised) [reply public/robots.txt:4]: Even better! Let me fix that up quickly.

Comment (anglepoised) [reply public/robots.txt]: OK, just killed the rule so we don't have a syntax issue.

**Pull request merged**